_sub/compute/helm-kube-prometheus-stack/vars.tf

```diff
@@ -1,3 +1,11 @@
+variable "aws_workload_account_id" {
```

Comment:
No type definition for this one?

Reply:
There's no type defined for it at the level it's passed in from (compute/k8s-services), and I kept it consistent with the _sub\compute\k8s-kiam submodule where it's also used in the same way.

That's not to say it shouldn't have a type set, or a description for that matter. I'll change it